app/models/grant_creator_request.rb

```diff
@@ -45,17 +45,15 @@ def requester_is_not_a_system_admin
   end
 
   def reviewer_is_a_system_admin
-    errors.add(:base, :reviewer_is_not_system_admin) if !reviewer.system_admin?
+    errors.add(:base, :reviewer_is_not_system_admin) unless reviewer.system_admin?
   end
 
-  def udpate_user_grant_creator_attribute
+  def update_user_grant_creator_attribute
     case status
     when 'approved'
       requester.update_attribute(:grant_creator, true)
-      GrantCreatorRequestReviewMailer.with(request: self).approved.deliver_now
     when 'rejected'
```

Comment:
seems redundant

Reply:
Requester's :grant_creator boolean is nil or false. Once approved, the requester's grant_creator attribute needs to be set to true.  Should this be somewhere else?

Reply:
sorry i wasn't being clear. I meant the case switch has two branches doing the same thing: 
`requester.update_attribute(:grant_creator, false)`, in both `when 'rejected'` and `else`. these can be combined.